Library/Homebrew/formula.rb

```diff
@@ -3126,7 +3126,9 @@ def eligible_kegs_for_cleanup(quiet: false)
   # or calling `do |staging| ... staging.retain!` in the block will skip
   # the deletion and retain the temporary directory's contents.
   def mktemp(prefix = name, opts = {}, &block)
-    Mktemp.new(prefix, opts).run(&block)
```

Comment:
Is it possible to convert to `kwargs` here too?

Reply:
Probably. It is currently private API, though we do use it in Homebrew/core so should be re-scoped to at least internal API.

Reply:
Yeh, I think it's worth changing in that case but agreed on API changes.